addons/common/functions/fnc_playMusicServer.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function plays synchronized music for all clients and prevents interruption. Supports serverside music queue.
+ *
+ * Arguments:
+ * 0: Music classname to play <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["LeadTrack01_F"] call afsk_common_fnc_playMusic
+ *
+ * Public: No
+ */
+
+params ["_musicClassname"];
+
+// Add to queue
+if GVAR(musicPlaying) exitWith {
+    GVAR(musicQueue) pushBack _musicClassname;
+};
```

Comment:
I don't see `GVAR(musicPlaying)` being set to true outside of EH so it seems that tracks will never be added to queue.
Also I don't see any playMusic being done on server outside of `MusicStop`, so will this EH even be fired?

Reply:
Yes, I totally forgot to include a way to play music for the first time. I think it is enough to replace `exitWith` with `then else` and use it to raise `playMusic` event.